server/plugin/utils_test.go

```diff
@@ -356,6 +356,41 @@ func TestGetPluginURL(t *testing.T) {
 	}
 }
 
+func TestSanitizeURLPath(t *testing.T) {
+	p := Plugin{}
+	for _, testCase := range []struct {
+		description  string
+		actualPath   string
+		expectedPath string
+	}{
+		{
+			description:  "SanitizeURLPath: valid",
+			actualPath:   "/dummy_org/dummy_project",
+			expectedPath: "/dummy_org/dummy_project",
+		},
+		{
+			description:  "SanitizeURLPath: invalid variation 1",
+			actualPath:   "/../dummy_org/dummy_project",
+			expectedPath: "/dummy_org/dummy_project",
+		},
+		{
+			description:  "SanitizeURLPath: invalid variation 2",
+			actualPath:   "../dummy_org/../dummy_project/..",
+			expectedPath: "/dummy_org/dummy_project",
+		},
+		{
+			description:  "SanitizeURLPath: invalid variation 3",
+			actualPath:   "../../dummy_org/dummy_project",
+			expectedPath: "/dummy_org/dummy_project",
```

Comment:
(question) is the path always a non-root one? (ie `abd/def` compared to `/abcd/ef'`)? Because if this is the case, would the extra slash break API calls? Did a manual test take place in the context of this change?

Reply:
there would not be extra slash as we have used `path.Clean` method which removes any extra slash left after removing `..` or `%2e%2e`
Yes, have tested manually Azure Devops APIs are working fine